### DISCUSSION TITLE
Airplay 2 Volume Control via HDMI-CEC on AVR ?

### DISCUSSION THREAD

**Discussion by PedroDeLaFrance** [mikebrady/shairport-sync#1784] [Q&A]
Hi !
I test shairport-sync, with mo0de, with a IQaudi DigiAmp+. It's great with my old Eltax Monitor 3, but too crystal sound. I preferd the sound with my AVR Marantz NR1504 (airplay 1 only, so no multiroom possibilities). So, I plug my Rpi 3b+, without the IQAudio HAT, with the HDMI cable on my AVR for playing Apple Music's in multiroom with Airplay 2 Shaireport-Sync capabilities. It's ok, I can modify the volume, but just the HDMI Output, not the master volume of the AVR. I would like to modify the master volume of the AVR instead when I modifiy the volume on airplay. Is it possible ? Like on my Apple TV ?

**Comment by mikebrady** [comment]
Thanks for the post. Shairport Sync can cause a script or application to be executed whenever the volume is changed. The question is whether a script could control the master volume of the AVR, and that really depends on the capabilities of the AVR itself — I’m afraid it’s beyond the scope of Shairport Sync.

**Reply by PedroDeLaFrance** [reply]
Thank's a lot for your reply. I wander how Apple do this, I think a standard must exist to do this.
